includes/admin/class-wc-stripe-inbox-notes.php

```diff
@@ -31,6 +32,24 @@ public function __construct() {
 		}
 	}
 
+	public static function create_upe_availability_note() {
+		$stripe_settings = get_option( 'woocommerce_stripe_settings', [] );
+		$stripe_enabled  = isset( $stripe_settings['enabled'] ) && 'yes' === $stripe_settings['enabled'];
+
+		/**
+		 * No need to display the admin inbox note when
+		 * - upe preview is disabled
+		 * - upe is already enabled
+		 * - stripe is not enabled
+		 */
+		if ( ! class_exists( 'WC_Stripe_UPE_Availability_Note', false ) || ! WC_Stripe_Feature_Flags::is_upe_preview_enabled() || WC_Stripe_Feature_Flags::is_upe_checkout_enabled() || ! $stripe_enabled ) {
+			return;
+		}
+
+		require_once WC_STRIPE_PLUGIN_PATH . '/includes/notes/class-wc-stripe-upe-availability-note.php';
```

Comment:
Do we need this line? If the class `WC_Stripe_UPE_Availability_Note` does not exist, it would have exited in line 45-46. So then if it goes through the condition there, this file is already loaded?